tenets/codelingo/effective-go/comment-first-word-as-subject/codelingo.yaml

```diff
@@ -0,0 +1,32 @@
+funcs:
+  - name: isValid
+    type: asserter
+    body: |
+      function(text, name) {
+        text = text.substring(2).trim()
+        var words = text.split(" ")
+        return words[0].toLowerCase() !== name.toLowerCase()
+      }
+tenets:
+  - name: comment-first-word-as-subject
+    flows:
+      codelingo/docs:
+        title: Comment First Word as Subject
+        body: |
+          Doc comments work best as complete sentences, which allow a wide variety of automated presentations. 
+          The first sentence should be a one-sentence summary that starts with the name being declared.
+      codelingo/review:
+        comment: |
+          The first sentence should be a one-sentence summary that starts with the name ({{funcName}}) being declared.
```

Comment:
We've been using `` for var names instead of brackets. e.g ...starts with func name `baz` being declared.